library/kani_core/src/lib.rs

```diff
@@ -54,7 +61,7 @@ macro_rules! kani_lib {
 /// TODO: Use this inside kani library so that we dont have to maintain two copies of the same intrinsics.
 #[macro_export]
 macro_rules! kani_intrinsics {
-    () => {
+    ($core:tt) => {
```

Comment:
I'm guess this still doesnt change how kani needs to be invoked from the std library or the user library?

Reply:
No... I don't think so. Users are supposed to invoke `kani!()` macro, and that hasn't changed.